packages/devtools/web/styles.css

```diff
@@ -765,6 +774,14 @@ div.tabnav-tab {
     height: 30px;
 }
 
+.form-select:hover {
+    background-color: var(--button-selected-color);
+}
+
+input {
+    cursor: pointer;
```

Comment:
A pointer as the cursor for input seems surprising. For example, I expect this isn't right for code mirror, etc.
Can we use some more specific css rules to use a pointer cursor where needed?

Reply:
good point. I did this to target checkboxes - i'll change to `input[type='checkbox']`